bgpd/bgp_network.c

```diff
@@ -57,8 +57,26 @@ struct bgp_listener {
 	union sockunion su;
 	struct thread *thread;
 	struct bgp *bgp;
+	char *name;
 };
 
+void bgp_dump_listener_info(struct vty *vty)
+{
+	struct listnode *node;
+	struct bgp_listener *listener;
+
+	vty_out(vty, "Name             fd Address\n");
+	vty_out(vty, "---------------------------\n");
+	for (ALL_LIST_ELEMENTS_RO(bm->listen_sockets, node, listener)) {
+		char buf[SU_ADDRSTRLEN];
+
+		vty_out(vty, "%-16s %d %s\n",
+			listener->name ? listener->name : VRF_DEFAULT_NAME,
```

Comment:
As per the new code in bgp_listener, listner->name never be NULL. Right ?

Reply:
yes

Reply:
Actually my comment is: as this never be NULL, why do we need this condition and so VRF_DEFAULT_NAME would never get printed.